core/sr-io/without_std.rs

```diff
@@ -385,6 +385,20 @@ pub mod ext {
 		/// - nonzero otherwise.
 		fn ext_submit_transaction(data: *const u8, len: u32) -> u32;
 
+		/// Returns information about the local node's network state.
+		///
+		/// # Returns
+		///
+		/// - The encoded `Result<offchain::LocalNetworkState, ()>`.
+		fn ext_local_network_state(msg_len: *mut u32) -> *mut u8;
+
+		/// Returns the locally configured authority public key, if available.
+		///
+		/// # Returns
+		///
+		/// - The encoded `Result<PublicKey encoded to Vec<u8>, ()>`.
```

Comment:
Why is the rationaly not using `Option`?

Reply:
I just used the same return type as the other existing functions in the API and would leave it like this for consistency.